src/frontend/src/flows/authenticate/index.ts

```diff
@@ -176,10 +80,12 @@ const pageContent = (
   </div>
   ${footer}`;
 
-const derivationOriginSection = (derivationOrigin: string) => html` <p>
+const derivationOriginSection = (derivationOrigin: string) => html` <p
+    class="t-paragraph"
+  >
     This service is an alias of:
   </p>
-  <div class="host-name highlightBox hostName">${derivationOrigin}</div>`;
+  <data class="c-input c-input--readonly t-vip"> ${derivationOrigin} </data>`;
```

Comment:
is that really a `data` element semantically speaking?

Reply:
generally speaking same question for all `<data />` element in this pr

Reply:
I thought this was one of those rare occasions where if actually made sense :D (The value that is in the innterHTML is the same that would be in the value="" attribute thats why I did not add the value attr). I am happy to change it to something else though. Any idea?

Reply:
PS: just realised that the value attribute must be present: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-data-element I am still open to suggestions about an adequate html element.

Reply:
I'm not sure it applies here typically because `derivationOrigin` is not a machine value per se but a configuration parameter set by the developer who uses II as auth to sign-in user in his/her dapp.

@mstrasinskis has often better idea than me in HTML semantic, do you have a cool idea and thought?

Reply:
Not sure about using the `data` tag here. If only there is a way/need to provide more readable friendly value in the  `value=...` attribute.